WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -925,6 +925,14 @@ class AbstractPostListViewController: UIViewController,
         updateFilterWithPostStatus(.scheduled)
     }
 
+    @objc func draftPost(_ apost: AbstractPost) {
```

Comment:
`draftPost` sounds like it will create a new post, imo. An action name like `moveToDraft` or `unpublish` or `turnIntoDraft`? Thoughts?

Reply:
Changed to `moveToDraft`.